src/utils.rs

```diff
@@ -52,6 +52,14 @@ pub fn random(n: u32) -> u32 {
     })
 }
 
+/// Returns given argument without changes.
+#[allow(dead_code)]
+#[doc(hidden)]
+#[inline(always)]
+pub(crate) fn identity<T>(arg: T) -> T {
+    arg
+}
+
```

Comment:
What is the difference between this function and [`std::convert::identity`](https://doc.rust-lang.org/std/convert/fn.identity.html)?

Reply:
Ah, thanks, I missed that this function was added a year ago already...